### PR TITLE
Update to 0.9.3

### PR DESCRIPTION
**Pull request by inglar**
https://github.com/sabre-io/Baikal/releases/tag/0.9.3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ I follow the same version naming scheme as [Baikal](http://sabre.io/baikal/) the
 
 The following tags support multiple architectures, e.g. `amd64`, `arm32v7`, `arm64v8` and `i386`.
 
+- [`0.9.3`, `0.9.3-apache`, `apache`, `latest`](https://github.com/ckulka/baikal-docker/blob/0.9.3/apache.dockerfile)
+- [`0.9.3-php8.0`, `0.9.3-apache-php8.0`, `apache-php8.0`, `latest-php8.0`](https://github.com/ckulka/baikal-docker/blob/0.9.3/apache-php8.0.dockerfile)
+- [`0.9.3-nginx`, `nginx`](https://github.com/ckulka/baikal-docker/blob/0.9.3/nginx.dockerfile)
+- [`0.9.3-nginx-php8.0`, `nginx-php8.0`](https://github.com/ckulka/baikal-docker/blob/0.9.3/nginx-php8.0.dockerfile)
 - [`0.9.2`, `0.9.2-apache`, `apache`, `latest`](https://github.com/ckulka/baikal-docker/blob/0.9.2/apache.dockerfile)
```
Comment (ckulka) [inline README.md:19]: `apache` and `latest` need to be removed from here, as those image tags come from 0.9.3 now

Comment (inglar) [reply README.md]: done

**Review by ckulka — APPROVED** [review]
Line 21 of `README.md` still has "nginx" which need to be removed, but I'll do that on the master branch myself. Thanks for providing the PR!

**Comment by ckulka** [comment]
The tests failed, something changed in the base nginx container which now makes `echo >&3 ...` fail... just FYI that I have to take care of that before I can publish 0.9.3

**Pull request merged**